app/models/user.rb

```diff
@@ -22,6 +22,7 @@ def toggle_enabled
   end
 
   def merchant_orders
+    []
     # return all orders where order contains items with user_id = @user.id
   end
```

Comment:
Need help with this method.  Should return all orders that contain items where user_id is equal to user instance id.  Relationship may be broken.  Need to add model method test!